vuecomponents/bundle/src/main/bundle/condition_eu.properties

```diff
@@ -0,0 +1,33 @@
+add_condition_as_prereq = Add condition as prerequisite
+add_condition = Add condition
+condition = Condition
+conjunction = Conjunction
+create_condition_for_this_item = Create new condition based on this item:
+display_the_item_completed = Require the item {0} to be completed.
+display_the_item_score = Require the item {0} to have a score {1} {2} points
+display_this_item_completed = Require this item to be completed.
+display_this_item_score = Require this item to have a score {0} {1} points.
+existing_conditions_for_this_item = Existing conditions based on this item:
+existing_prereq_conditions_and = Conditions selected as prerequisite for this item that all need to be fullfilled (AND):
+existing_prereq_conditions_or = Conditions selected as prerequisite for this item where one needs to be fullfilled (OR):
+form_require_item_points = Require this item to have a score
+no_condition_to_pick = No condition from other item available to select
+pick_condition_as_prereq = Select a condition as a prerequisite for this item:
+points = Points
+remove_condition = Remove condition
+removing_condition = Removing condition
+saving_condition = Saving condition
+saving_condition_as_prereq = Saving condition as prerequisite
+tag_in_use = in use
+tag_unused = unused
+unknown_condition = Condition of unknown type
+
+# Operators
+smaller_than = smaller than
+smaller_than_or_equal_to = smaller than or equal to
+equal_to = equal to
+greater_than_or_equal_to = greater than or equal to
+greater_than = greater than
+
+conjunction_and = Require this condition AND another condition
+conjunction_or = Require this condition OR another condition
```

Comment:
Hi @PabloSanRoman!
Could you please help me with this translations, when you have time? No rush.
I leave some screenshots here for context:

![image](https://github.com/sakaiproject/sakai/assets/53173679/15c9d29b-0929-4227-9b00-0c4d15407611)
![image](https://github.com/sakaiproject/sakai/assets/53173679/9c8880b8-1f74-4e21-9c0f-5fabdc0b2232)
![image](https://github.com/sakaiproject/sakai/assets/53173679/476fcd9b-197e-45c6-a55d-e423a1ec42a8)